src/chains/Osmosis/index.ts

```diff
@@ -5,12 +5,17 @@ import { bech32 } from "bech32";
 export default class Osmosis extends Axelar implements Chain {
   public chainInfo: ChainInfo = {
     chainSymbol: "OSMO",
-    chainName: "Osmosis-2",
+    chainName: "Osmosis",
     estimatedWaitTime: 5,
     fullySupported: false,
     assets: [],
     txFeeInPercent: 0.1,
     module: "axelarnet",
+    chainIdentifier: {
+      devnet: "osmosis",
+      testnet: "osmosis-2",
```

Comment:
I didn't expect we can fix this, but why osmosis uses different `chainId` accross networks 😩

Reply:
This is the identifier on our network. Due to an osmosis testnet issue, we had to re-add the osmosis chain under a new name.